cluster_library.c

```diff
@@ -1754,6 +1754,18 @@ PHP_REDIS_API void cluster_ping_resp(INTERNAL_FUNCTION_PARAMETERS, redisCluster
     CLUSTER_RETURN_BOOL(c, 1);
 }
 
+PHP_REDIS_API void
+cluster_pop_resp(INTERNAL_FUNCTION_PARAMETERS, redisCluster *c, void *ctx)
+{
+    if (ctx == NULL) {
+        cluster_bulk_resp(INTERNAL_FUNCTION_PARAM_PASSTHRU, c, NULL);
+    } else if (ctx == PHPREDIS_CTX_PTR) {
+        cluster_mbulk_raw_resp(INTERNAL_FUNCTION_PARAM_PASSTHRU, c, NULL);
+    } else {
+        ZEND_ASSERT(!"memory corruption?");
```

Comment:
```suggestion
        ZEND_ASSERT("memory corruption?");
```

Reply:
We need `false` condition to trigger zeng_assert